.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -20,19 +20,19 @@ Fedora packaging:
  - [ ] Run `kinit your_fas_account@FEDORAPROJECT.ORG`
  - [ ] Run `fedpkg new-sources tarball-name`
  - [ ] PR the changes in [Fedora](https://src.fedoraproject.org/rpms/ignition)
- - [ ] Once the PR merges to rawhide, merge rawhide into the other relevant branches (e.g. f35) then push those, for example:
+ - [ ] Once the PR merges to rawhide, merge rawhide into the other relevant branches (e.g. f36) then push those, for example:
    ```bash
    git checkout rawhide
    git pull --ff-only
-   git checkout f35
+   git checkout f36
    git merge --ff-only rawhide
-   git push origin f35
+   git push origin f36
    ```
- - [ ] On each of those branches run `fedpkg build`
+ - [ ] On each of those branches run `fedpkg build` including rawhide.
```

Comment:
Did you intend to have this in https://github.com/coreos/butane/pull/349 also?

Reply:
Yeah, I should add it there as well.

Reply:
done